app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -70,11 +71,25 @@ def authorization_params
       params.permit(OpenidConnectAuthorizeForm::ATTRS)
     end
 
+    def handle_invalid_logged_out_form
+      return if current_user
```

Comment:
Remove this line, so that the request is always validated? For example, what if the user is already signed in, but then makes a bad OIDC request? Should we add a test for that scenario to make sure it doesn't blow up?